Lerc/src/Lerc1Decode/CntZImage.cpp

```diff
@@ -68,6 +68,17 @@ unsigned int CntZImage::computeNumBytesNeededToReadHeader()
 
 // -------------------------------------------------------------------------- ;
 
+#if defined(RTC_COCOA_OSX)

```

Comment:
should this also include `&& !defined(DEBUG)`?

Reply:
Yep, good catch.